modules/installation-user-infra-generate-k8s-manifest-ignition.adoc

```diff
@@ -86,6 +103,14 @@ The installation configuration file transforms into the Kubernetes manifests. Th
 The Ignition config files that the {product-title} installation program generates contain certificates that expire after 24 hours, which are then renewed at that time. If the cluster is shut down before renewing the certificates and the cluster is later restarted after the 24 hours have elapsed, the cluster automatically recovers the expired certificates. The exception is that you must manually approve the pending `node-bootstrapper` certificate signing requests (CSRs) to recover kubelet certificates. See the documentation for _Recovering from expired control plane certificates_ for more information.
 ====
 
+ifdef::ibm-z[]
+[NOTE]
+====
+The installation program that generates the manifest and Ignition files is architecture specific and can be obtained from the
+link:https://mirror.openshift.com/pub/openshift-v4/s390x/clients/ocp/latest/[client image mirror]. The Linux version of the installation program runs on s390x only. This installer program is also available as a Mac OS version.
+====
+endif::ibm-z[]
+
```

Comment:
ifdef::ibm-power[]
[NOTE]
====
The installation program that generates the manifest and Ignition files is architecture specific and can be obtained from the 
link:https://mirror.openshift.com/pub/openshift-v4/ppc64le/clients/ocp/latest/[client image mirror]. The Linux version of the installation program runs on ppc64le only. This installer program is also available as a Mac OS version.
====
endif::ibm-power[]